.travis.yml

```diff
@@ -23,13 +41,20 @@ install:
   # cabal and ghc tries to use them all. Which is bad idea on a shared box.
   # See also: https://ghc.haskell.org/trac/ghc/ticket/9221
   - sed -i 's/^jobs:/-- jobs:/' ${HOME}/.cabal/config
-  - mkdir -p sandboxes/$GHCVER/${STACKAGE:-none}
-  - cabal sandbox init --sandbox sandboxes/$GHCVER/${STACKAGE:-none}
-  # Download stackage cabal.config
-  # Not sure whether filtering out purescript is necessary
+  # Cache sandboxes in ~/cabal-sandboxes
+  # Move right sandbox to .cabal-sandbox if exists
+  - if [ -d ~/cabal-sandboxes/$GHCVER-${STACKAGE:-none} ]; then
```

Comment:
Won't this share the sandbox between the main build and the `sdist` test? @hdgarrood Is that ok?